### DISCUSSION TITLE
[Feature suggestion] Pinterest plug-in for automatically changing the wallpaper using selected album

### DISCUSSION THREAD

**Discussion by Yoddikko** [koreader/koreader#12552] [Ideas]
Just an idea, I could also try to work on this

**Comment by Frenzie** [comment]
It wouldn't really need to have anything to do with KOReader, but you could write it as a KOReader plugin. You can already set a specific image or a random image from a folder, so you'd just need to update the specific image or the images in a folder.

**Reply by Yoddikko** [reply]
Good to know! I'll onstall KOReader on my kindle this night and make some tests. Thanks so much!
